### DISCUSSION TITLE
how to  import plotly

### DISCUSSION THREAD

**Discussion by galsang-git** [observablehq/framework#1869] [Q&A] [answered]

**Comment by Fil** [comment, marked as answer]
~~~~
```js
import Plotly from "npm:plotly.js-dist-min@2.35.2";
```
~~~~
see https://observablehq.observablehq.cloud/pangea/party/plotly for an example

**Reply by Fil** [reply]
Note: I've just updated the code above to pin the version of plotly, because the `@latest` version of plotly currently (`@2.35.3`) fails to load.
